docs/conf.py

```diff
@@ -20,15 +20,19 @@
 
 
 def init_tutorials():
-    if not "conf.py" in os.listdir():
+    if "conf.py" not in os.listdir():
         raise FileNotFoundError("Wrong directory...")
 
-    if not TUTREPO in os.listdir():
+    if TUTREPO not in os.listdir():
         command = f"git clone {TUTURL}"
         print(command)
         subprocess.call(command, shell=True)
```

Comment:
use `subprocess.run(command, shell=True, check=True)`
`subprocess.call` is from python <= 3.5
check=True raises exception on error